dev/core/lis-modal-element.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>LIS Web-Components - &lt;lis-modal-element&gt;</title>
+  <!-- CSS framework -->
+  <link rel="stylesheet" type="text/css" href="../../node_modules/uikit/dist/css/uikit.min.css">
+  <script src="../../node_modules/uikit/dist/js/uikit.min.js"></script>
+  <!-- web components polyfills -->
+  <script src="../../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+  <script src="../../node_modules/lit/polyfill-support.js"></script>
+  <!-- GraphQL -->
+  <script type="text/javascript" src="../graphql.js"></script>
+  <!-- web components 
+  <script type="module" src="../../dist/index.js"></script>-->
+  <script type="module" src="../../dist/web-components.bundled.js"></script>
+</head>
+
+<body>
+
+  <div class="uk-container">
+    <h1>&lt;lis-modal-element&gt;</h1>
+    <p>
+      The <code>&lt;lis-modal-element&gt;</code> provides a generic component for rendering a modal.
+      In this example, the modal is populated with the lis-simple-table-element web component.
```

Comment:
Put lis-simple-table-element in `<code>` tags like lis-modal-element is.

Also mention that the element is a wrapper for the UIkit modal and that you interact with the modal using the UIkit API (provide a link) and the modal-id property.